src/transport/raw/GroupAddress.h

```diff
@@ -0,0 +1,69 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ * @brief
+ *    File contains definitions on how a connection to a peer can be defined.
+ *
+ */
+
+#pragma once
+
+// #include <stdio.h>
+// #include <inet/IPAddress.h>
+// #include <inet/InetInterface.h>
+// #include <lib/core/CHIPConfig.h>
+#include "PeerAddress.h"
+#include <app/util/basic-types.h>
+
+namespace chip {
+namespace Transport {
+
+/**
+ * Describes how a group on a CHIP network can be addressed.
+ * See 2.5.6.2. IPv6 Multicast Address
```

Comment:
```suggestion
 * See 2.5.6.2. IPv6 Multicast Address
 *
 * NOTE that the Fabric ID is used directly, without any scoping. Therefore,
 * these addresses do not fully identify which Fabric they come from since
 * the root of trust identifier is not used, unlike in compressed fabric references
 * used for operational discovery. This means the same GroupAddress may
 * alias between fabrics, but this is OK since the goal is only to filter traffic to
 * subscribed groups. Group key security ensures that even if you receive a
 * message from a different fabric due to FabricID aliasing, it will be ignored or
 * rejected by non-members.
```